lib/builtin.pm

```diff
@@ -0,0 +1,93 @@
+package builtin;
+
+use strict;
+use warnings;
+
+# All code, including &import, is implemented by always-present functions in
+# the perl interpreter itself.
+# See also `builtin.c` in perl source
+
+1;
+__END__
+
+=head1 NAME
+
+builtin - Perl pragma to import built-in utility functions
+
+=head1 SYNOPSIS
+
+    use builtin qw( true false isbool );
+
+=head1 DESCRIPTION
+
+Perl provides several utility functions in a package called C<builtin::>.
+Functions in this namespace can be called by their fully-qualified names, but
+it is often desired to give them shorter name aliases for convenience.
+
+Individual named functions can be imported by listing them as import
+parameters on the C<use> statement for this pragma.
+
+=head2 Lexical Import
+
+This pragma module creates I<lexical> aliases in the currently-compiling scope
+to these builtin functions.
```

Comment:
Might be helpful context to mention that it operates similarly to keywords provided by the feature pragma.